### DISCUSSION TITLE
Closed and merged PRs both use purple icon in issue timeline

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#6767] [Projects and Issues] [answered]
Related to: https://github.blog/changelog/2021-10-26-updates-to-our-issue-status-icons-and-colors/

[Closed PR example](https://github.com/Cog-Creators/Red-DiscordBot/pull/5423)
![image](https://user-images.githubusercontent.com/6032823/138936109-73c5c0d9-43fb-4769-b2a5-5095efd1bc4d.png)

[Merged PR example](https://github.com/Cog-Creators/Red-DiscordBot/pull/5412)
![image](https://user-images.githubusercontent.com/6032823/138935991-4c5a8480-645d-4b94-a3b0-e02efb1ee68a.png)

It is quite confusing. Perhaps using gray here would be better?

**Comment by haykam821** [comment]
I liked being able to differentiate between closed and merged from notifications easily. Could this be an option? I'd rather revert to open being green, closed being red, and merged being purple. Having gray color would decrease contrast, which would also be detrimental, in addition to conflicting with drafts.

**Comment by Jolg42** [comment]
I think many people were surprised, myself included. I really dislike this right now as it makes it hard to know from first sight if it's an issue or a PR.

**Reply by Jolg42** [reply]
Related https://github.com/github/feedback/discussions/6795

**Comment by Katsute** [comment]
I've created a chrome extension if you want to revert the timeline back to red/green: <https://github.com/Katsute/GitHub-Red-Closed-Issues-Extension>

**Comment by JannesPeters** [comment, marked as answer]
This should be fixed now, and closed PRs are shown as red in the timeline
